mir_eval/separation.py

```diff
@@ -52,9 +52,9 @@ def validate(reference_sources, estimated_sources):
 
     Parameters
     ----------
-    reference_sources : np.ndarray, shape=(nsrc, nsampl)
+    reference_sources : np.ndarray, shape=(nsrc1, nsampl1)
```

Comment:
Well, you could make the same argument for all of the metrics functions themselves - we have not verified them either.  To me, the docstring typically specifies "What input to this function is expected and will not cause an exception?" which is the convention through `mir_eval`.